### PR TITLE
Updating package names to discussed naming conventions

### PR DESCRIPTION
**Pull request by pavolum**
### Purpose
Updating the package names and versions for generator-conversational-core, generator-adaptive-bot and generator-empty-bot to scoped Microsoft namespace per discussion with the larger team on desired naming for R12 release.

### CODE REVIEW

File: generators/generator-conversational-core/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "generator-conversational-core",
-  "version": "1.0.12",
+  "name": "@microsoft/generator-microsoft-bot-conversational-core",
+  "version": "1.0.0-preview1",
```
Comment (ryanlengel) [inline generators/generator-conversational-core/package.json:3]: You can make this one "preview.1" as well

Comment (peterinnesmsft) [reply generators/generator-conversational-core/package.json:3]: For my benefit, are we going with preview.1, or preview1? I don't mind either way, but I just changed them in the other components to preview1. I don't mind either way, but now would be the time to make the call and pull the trigger.

**Review by peterinnesmsft — CHANGES_REQUESTED** [review]
Can you rename corresponding directory structures and whatnot to reflect updated package names?

**Comment by pavolum** [comment]
starting a new PR to handle merge conflicts

**Pull request closed without merging**